app/views/users/_form.html.erb

```diff
@@ -1,51 +1,49 @@
 <div style="padding:10px;">
-<%= form_for @user, :url => {:controller => 'users', :action => @action}, :html => {:class => "row"} do |f| %>
-
+  <%= form_for :user, :as => :user, :url => :users, :html => {:class => "form col-sm-12 offset-3"} do |f| %>
```

Comment:
So, the line needs to be 
<%= form_for @user, :url => :register , :html => {:class => "row"} do |f| %>

Reply:
great, I'll try out both. Thanks